exercises/concept/city-office/.docs/instructions.md

```diff
@@ -10,38 +10,46 @@ You decide to write some documentation so that it's easier for your new colleagu
 
 Add documentation to the `Form` module that describes its purpose. It should read:
 
-> A collection of loosely related functions helpful for filling out various forms at the city office.
+```
+A collection of loosely related functions helpful for filling out various forms at the city office.
+```
 
 ## 2. Document filling out fields with blank values
 
 Add documentation and a typespec to the `Form.blanks/1` function. The documentation should read:
 
-> Generates a string of a given length.
->
-> This string can be used to fill out a form field that is supposed to have no value.
-> Such fields cannot be left empty because a malicious third party could fill them out with false data.
+```
+Generates a string of a given length.
+
+This string can be used to fill out a form field that is supposed to have no value.
+Such fields cannot be left empty because a malicious third party could fill them out with false data.
+```
```

Comment:
Those blocks showing the sting that you need to copy into `@doc` cannot be quotes because quotes do not keep the original whitespace and newlines were missing in the output.